IOSAccessAssessment/IOSAccessAssessmentApp.swift

```diff
@@ -9,20 +9,16 @@ import SwiftUI
 
 @main
 struct IOSAccessAssessmentApp: App {
-    private let keychainService = KeychainService()
-    @State private var isAuthenticated: Bool
-    
-    init() {
-        let isTokenValid = keychainService.isTokenValid()
-        _isAuthenticated = State(initialValue: isTokenValid)
-    }
+    @StateObject private var userState = UserStateViewModel()
```

Comment:
Oh I didn't notice this while reviewing the login PR. 
This does help quite a few things. Nice one!

Reply:
I am having some second thoughts on if ViewModel is something we should go ahead with, after having most of the application being implemented in (sort of) MV pattern.

Reply:
This thread is mostly just a thought experiment I'd like to discuss whenever possible. 
I won't be blocking the PR just for this.